mec/cmd/mec/main.go

```diff
@@ -95,7 +94,10 @@ func createCommand(args []string) *cobra.Command {
 				log.Errorf("Failed to create OSSMPullStrategy: %v", err)
 			}
 			w := server.NewWorker(config, p, baseURL, serveDirectory)
-
+			if w == nil {
+				log.Errorf("Failed to create worker: %v", err)
```

Comment:
What's the error you're reporting here? `server.NewWorker` doesn't return an error. Looks like that would be the error for `NewOSSMPullStrategy` since that doesn't return on error?

Reply:
good catch. will fix